src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -203,6 +203,38 @@ import { ViewTransitions } from 'astro:transitions';
 The `morph` animation cannot be simulated in traditional CSS. So any element using this animation will not be animated.
 :::
 
+## Script behavior during page navigation
+
+When navigating between pages with the `<ViewTransitions />` component, scripts are run in sequential order to match browser behavior. 
+
+If you have code that sets up global state, this state will need to take into account that the script might execute more than once. Check for the global state in your `<script>` tag, and conditionally execute your code where possible:
+
+```astro
+<script is:inline>
+  if(!window.SomeGlobal) {
+    window.SomeGlobal = {} // ....
+  }
+</script>
+```
+
+Module scripts are only ever executed once because the browser keeps track of which modules are already loaded. For these scripts, you do not need to worry about re-execution.
+
+#### `astro:load`
+
+An event that fires on page navigation. You can listen to this event on the `document`.
+
+The `<ViewTransitions />` component fires this event both on initial page navigation (MPA) and any subsequent navigation, either forwards or backwards.
+
+You can use this event to run code on every page navigation, or only once ever:
+
+```astro
```

Comment:
This addition looks great!

One small suggestion to highlight the `once` argument in this code block to reinforce that it’s there. Given this is the only example we give of using `astro:load` I’m a little concerned some folks may just blindly copy without noticing the `once`:

````suggestion
```astro "{ once: true }"
````